crates/ide/src/syntax_highlighting/tests.rs

```diff
@@ -1126,5 +1126,5 @@ fn benchmark_syntax_highlighting_parser() {
             .filter(|it| it.highlight.tag == HlTag::Symbol(SymbolKind::Function))
             .count()
     };
-    assert_eq!(hash, 1609);
+    assert_eq!(hash, 1608);
```

Comment:
This *somehow* affects the outcome here, yet the parse trees don't differ on my machine 😕